### ISSUE TITLE
Add option to exclude backup folder with subfolders backups selected

### ISSUE THREAD

**Issue by reelpilot** [smeighan/xLights#3193]
**Is your feature request related to a problem? Please describe.**
When 'backup subfolders' is selected in the settings, xlights creates backups of all subfolders including the backup folder. This means each backup folder that xlights generates contains its own backup folder inside which then contains all the previous backups. It's like looking into an infinity mirror of backup files that wastes space and leads to excessive file syncing if using cloud storage like dropbox. 

**Describe the solution you'd like**
I'm not sure if there is really a need to even make a copy of the backup folder when backing up subfolders in the first place as each backup made contains all the files one would ever really need. So I think xlights should either omit the backup folder when 'backup subfolders' is selected or have a second checkbox to not include the backup folder when backing up subfolders.

**Comment by keithsw1111** [comment]
There is already code there to prevent it backing up backup folders ... are you able to create a simple folder structure that shows the problem and share it.

**Comment by reelpilot** [comment]
I discovered this the other day when getting xlights setup on a new Macbook. On my older windows machine I had backup subfolders disabled, but I guess that it was enabled by default when I installed xlights on the new Macbook. Since you mention that it should not be doing this in the first place, I did a little test and found that it is only backing up the backups folder on this one directory. I cannot get it to duplicate the issue when I use a different xlights directory. I can make a short video showing the problem and export a quick sequence so you'll have the files.
